src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -521,7 +521,7 @@ public override void Write(byte[] buffer, int offset, int size)
 
         private int _closeTimeout = Socket.DefaultCloseTimeout; // 1 ms; -1 = respect linger options
 
-        public void Close(int timeout)
+        internal void Close(int timeout)
```

Comment:
Is there other internal code that is still using these methods? If not, let's just delete them.